### DISCUSSION TITLE
Running the example in a separate project - refuses to work, why?

### DISCUSSION THREAD

**Discussion by KoehlerT** [aevyrie/bevy_mod_picking#245] [Q&A] [answered]
I am trying to get this library to work in my own project, but somehow I cannot even get the example to work.

I have created a new project, copy pasted the examples/gltf.rs to main.rs.
```
.
├── Cargo.toml
├── assets
│   ├── images
│   └── models
│       └── FlightHelmet
└── src
    └── main.rs
```
My `Cargo.toml` looks like this:
```
[package]
name = "select_test"
version = "0.1.0"
edition = "2021"
[dependencies]
bevy = "0.11"
bevy_eventlistener = "0.3.0"
bevy_mod_picking = "0.15.0"

[profile.dev]
opt-level = 1
[profile.dev.package."*"]
opt-level = 3
```
I have copy pasted the assets folder. Using `cargo run` executes the app, the helmet is shown, but highlighting/selection does not work. Do you have a hint where the problem may lie?
`cargo run --example gltf` works btw.
I am a bit out of idea and would really appreciate if anyone had an idea what this stupid error could be.

**Comment by Froidoh** [comment]
I have the exact same problem!

**Comment by mogambro** [comment]
I can confirm that it doesn't work with version 0.15 in another project.
If I checkout v0.15.0 from here though, it works.

If, for the copied version, I use the git version though, it works like a charm.

```
bevy_mod_picking = { git = "https://github.com/aevyrie/bevy_mod_picking" } 
```
Not sure what's going on, after checking the commits that are not in v0.15.0

@aevyrie Is a new release needed maybe?

**Comment by aevyrie** [comment, marked as answer]
The issue is you are copying the example from `main`, which is not compatible with `v0.15.0`, there have been breaking changes since that release. You need to copy the example from the `v0.15.0` tag:

https://github.com/aevyrie/bevy_mod_picking/blob/v0.15.0/examples/gltf.rs

The difference is that raycast markers are no longer required, which means that using the example from `main` is not going to work on the `0.15` release of the plugin. See the changelog notes here:

https://github.com/aevyrie/bevy_mod_picking/commit/42718da9070a17a5ca358ea5f4a5894797e2ed0d#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR3-R7

These are the lines that were removed on main:
https://github.com/aevyrie/bevy_mod_picking/commit/42718da9070a17a5ca358ea5f4a5894797e2ed0d#diff-ec24d799681fb63c7d74bebbea39f99b1782f5a4c84cd7ece0c147c418274477
